checkov/common/sca/output.py

```diff
@@ -111,14 +114,17 @@ def _update_details_by_scan_data_format(
 
 
 def create_report_cve_record(
-    rootless_file_path: str,
-    file_abs_path: str,
-    check_class: str,
-    vulnerability_details: dict[str, Any],
-    licenses: str,
-    runner_filter: RunnerFilter | None = None,
-    sca_details: SCADetails | None = None,
-    scan_data_format: ScanDataFormat = ScanDataFormat.TWISTCLI
+        rootless_file_path: str,
+        file_abs_path: str,
+        check_class: str,
+        vulnerability_details: dict[str, Any],
+        licenses: str,
+        root_package_alias: str,
+        root_package_version: str,
+        root_package_name: str,
```

Comment:
```suggestion
        root_package_alias: str| None = None,
        root_package_version: str| None = None,
        root_package_name: str| None = None,
```
twistlock scan for example didnt have dependencies and these fields should be an optional for this kind of scan